examples/tour.jl

```diff
@@ -30,58 +32,86 @@ using Onda, Legolas, Arrow, TimeSpans, DataFrames, Dates, UUIDs, Test
+# sample data files to a temporary directory. Onda signals are represented via the Legolas-generated record
+# type `SignalV2`, but are often initially constructed via `Onda`'s `store` method as seen below.
+
+saw_waves(info, duration) = [(j + i) % 100 * info.sample_resolution_in_unit for
+                             i in 1:channel_count(info), j in 1:sample_count(info, duration)]
+
+signals = SignalV2[]
 signals_recordings = [uuid4() for _ in 1:2]
 for recording in signals_recordings
-    for (kind, file_format, channels) in (("eeg", "lpcm", ["fp1", "f3", "c3", "p3",
-                                                           "f7", "t3", "t5", "o1",
-                                                           "fz", "cz", "pz",
-                                                           "fp2", "f4", "c4", "p4",
-                                                           "f8", "t4", "t6", "o2"]),
-                                           ("ecg", "lpcm.zst", ["avl", "avr"]),
-                                           ("spo2", "lpcm", ["spo2"]))
-        file_path = joinpath(root, string(recording, "_", kind, ".", file_format))
+    for (sensor_type, file_format, channels) in (("eeg", "lpcm", ["fp1", "f3", "c3", "p3",
+                                                                  "f7", "t3", "t5", "o1",
+                                                                  "fz", "cz", "pz",
+                                                                  "fp2", "f4", "c4", "p4",
+                                                                  "f8", "t4", "t6", "o2"]),
+                                                 ("ecg", "lpcm.zst", ["avl", "avr"]),
+                                                 ("spo2", "lpcm", ["spo2"]))
+        file_path = joinpath(root, string(recording, "_", sensor_type, ".", file_format))
         Onda.log("generating $file_path...")
-        info = SamplesInfo(; kind=kind, channels=channels,
-                           sample_unit="microvolt",
-                           sample_resolution_in_unit=rand((0.25, 1)),
-                           sample_offset_in_unit=rand((-1, 0, 1)),
-                           sample_type=rand((Float32, Int16, Int32)),
-                           sample_rate=rand((128, 256, 143.5)))
-        data = saws(info, Minute(rand(1:10)))
+        info = SamplesInfoV2(; sensor_type=sensor_type, channels=channels,
+                             sample_unit="microvolt",
+                             sample_resolution_in_unit=rand((0.25, 1)),
+                             sample_offset_in_unit=rand((-1, 0, 1)),
+                             sample_type=rand((Float32, Int16, Int32)),
+                             sample_rate=rand((128, 256, 143.5)))
+        data = saw_waves(info, Minute(rand(1:10)))
         samples = Samples(data, info, false)
         start = Second(rand(0:30))
         signal = store(file_path, file_format, samples, recording, start)
```

Comment:
good suggestion

and yeah, I noticed this as I was working on it - this intro itself could also be rewritten to have a far more gradual on-ramp

semi-relatedly - as Legolas evolves to be more and more generically useful, the expectation on downstream users to generically know Legolas in a DSL-ish sense also grows in tandem. The curse of DSL-like stuff lol. I guess this is inherently difficult to avoid (and/or a non-goal, since we do genuinely want folks to learn the underlying tool) but it definitely contributes to tours like these getting a bit firehose-y (or otherwise redundant with generic Legolas docs)